traefik/values.yaml

```diff
@@ -908,6 +908,8 @@ ports:
         allowEncodedPercent: false
         allowEncodedQuestionMark: false
         allowEncodedHash: false
+      # -- Maximum size of request headers in bytes. Default: 1048576 (1 MB)
+      maxHeaderBytes:  # @schema type:[string, null]
```

Comment:
Either use type `integer` or use  `traefik.convertMemToBytes` to make it easier for users.

Reply:
`integer` or `number` convert _2097152_ to _2.097152e+06_. 
I'm unsure this is what we want :thinking: